src/styles/Avenir/fonts.module.scss

```
@@ -0,0 +1,17 @@
+@font-face {
+    font-family: 'Avenir-Medium';
+    src: local('Avenir Medium'), local('Avenir-Medium'),
+        url('styles/Avenir/Avenir-Medium.woff2') format('woff2'),
+        url('styles/Avenir/Avenir-Medium.woff') format('woff');
+    font-weight: 500;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Avenir-Book';
+    src: local('Avenir Book'), local('Avenir-Book'),
+        url('styles/Avenir/Avenir-Book.woff2') format('woff2'),
+        url('styles/Avenir/Avenir-Book.woff') format('woff');
+    font-weight: 450;
+    font-style: normal;
+}

```

мне кажется, тебе будет удобнее не указывать под каждую жирность свою font-family.. ну типо это вроде бессмысленно и неудобно это делать, хотя мб чего-то не замечаю..
```suggestion
@font-face {
    font-family: 'Avenir';
    src: local('Avenir Medium'), local('Avenir-Medium'),
        url('styles/Avenir/Avenir-Medium.woff2') format('woff2'),
        url('styles/Avenir/Avenir-Medium.woff') format('woff');
    font-weight: 500;
    font-style: normal;
}

@font-face {
    font-family: 'Avenir';
    src: local('Avenir Book'), local('Avenir-Book'),
        url('styles/Avenir/Avenir-Book.woff2') format('woff2'),
        url('styles/Avenir/Avenir-Book.woff') format('woff');
    font-weight: 450;
    font-style: normal;
}

```